src/groups.json

```diff
@@ -6,5 +6,6 @@
   "activities",
   "objects",
   "symbols",
-  "flags"
+  "flags",
+  "custom"
```

Comment:
This list is generated dynamically on build from the emojis package. "custom" will then be removed. This file shouldn't be modified manually.
```suggestion
```

Reply:
removed on commit 1ba34bfb76ecc6a9a388036b166c160ac507c289